test/e2e/common.go

```diff
@@ -7,63 +7,94 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// newPod returns a new Pod object.
-func newPod(namespace string, name string, containerName string, runtimeclass string) *corev1.Pod {
-	return &corev1.Pod{
-		ObjectMeta: metav1.ObjectMeta{Name: name, Namespace: namespace},
-		Spec: corev1.PodSpec{
-			Containers:       []corev1.Container{{Name: containerName, Image: "nginx"}},
-			DNSPolicy:        "ClusterFirst",
-			RestartPolicy:    "Never",
-			RuntimeClassName: &runtimeclass,
-		},
+type RuntimeClass string
```

Comment:
I think there is a question whether we need any runtimeclasses other than `kata-remote` at this point as with the latest operator install changes that's the only runtimeClass we install now?

Reply:
I removed the runtime classes and use the fixed `kata-remote` runtime class everywhere.